test/unit/Editing.test.ts

```diff
@@ -0,0 +1,33 @@
+import './editing-element.js';
+import { EditingElement } from './editing-element.js';
+import { newActionEvent, newOpenDocEvent } from '../../src/foundation.js';
+import { html, fixture, expect } from '@open-wc/testing';
+
+describe('Editing Element', () => {
+  let elm: EditingElement;
+  const doc = new DOMParser().parseFromString(
+    `<?xml version="1.0" encoding="UTF-8"?>
+      <SCL version="2007" revision="B" xmlns:sxy="http://www.iec.ch/61850/2003/SCLcoordinates" xmlns="http://www.iec.ch/61850/2003/SCL" xmlns:txy="http://www.iec.ch/61850/2003/Terminal" xmlns:scl="http://www.iec.ch/61850/2003/SCL" xsi:schemaLocation="http://www.iec.ch/61850/2003/SCL SCL.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:IEC_60870_5_104="http://www.iec.ch/61850-80-1/2007/SCL">
```

Comment:
This SCL element does not need all the namespace definitions. Here `xmlns="http://www.iec.ch/61850/2003/SCL"` is more than enough.